Packs/CimTrak-SystemIntegrityAssurance/Playbooks/playbook-CimTrak_-_Example_-_Scan_Compliance_By_IP.yml

```diff
@@ -0,0 +1,165 @@
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 25043b93-4265-49a0-8858-04aed0ec4edd
+    propagationLabels:
+    - all
+    toServerVersion: ""
+id: a28a5a93-7020-4221-82de-bfc9ec85a86d
```

Comment:
@kigerjoel Playbook id should be the same as its name.

Reply:
Fixed